brightener.cpp

```diff
@@ -1,16 +1,8 @@
 #include "./brightener.h"
 #include <utility>
 
-ImageBrightener::ImageBrightener(std::unique_ptr<Image> inputImage)
-    : m_inputImage(std::move(inputImage)) {
-}
-
-bool ImageBrightener::ValidateImage() {
-    if (m_inputImage->m_columns <= 1024 && m_inputImage->m_rows <= 1024) {
-        return true;
-    } else {
-        return false;
-    }
+ImageBrightener::ImageBrightener(std::shared_ptr<Image> inputImage)
+    : m_inputImage(inputImage) {
 }
 
```

Comment:
Are you having trouble reducing the complexity here?
Some ways you can approach it:
- separate the processing (brightening) from the iteration (for loop)
- this does not require row and column distinctions. So you can iterate through the whole array in a single loop as well